sdk/core/System.ClientModel/tests/client/Internal/ModelReaderWriterExtensions.cs

```diff
@@ -0,0 +1,264 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
```

Comment:
Are we concerned that some of these internal types enable nullablility and some disable it?  Should we be consistent in our approach within these files?

Reply:
The goal was to be consistent with the actual shared source that will be used in our implementation.  I didn't want to differ from that.